### PR TITLE
feat : cart & wishlist

### PR DESCRIPTION
**Pull request by sudipkundu999**

### CODE REVIEW

**Review by archit7448 — APPROVED** [review]
Nice work good to merge , proper naming

File: pages/wishlist/wishlist.html [hunk 1]
```diff
@@ -5,8 +5,138 @@
...
+            <span class="card-text-overlay">
+                Out of stock!
+            </span>
+            <span class="card-badge">New</span>
+            <div class="card-image-container">
+                <button class="card-favourite">
+                    <i class="far fa-heart fa-2x"></i>
+                </button>
+                <img src="/assets/alchemist.jpeg" alt="">
+            </div>
```
Comment (srishtigautam21) [inline pages/wishlist/wishlist.html:113]: you should write something in alt

**Review by srishtigautam21 — APPROVED** [review]
good to merge

File: pages/wishlist/wishlist.html (hunk 1 above)
Comment (sudipkundu999) [reply pages/wishlist/wishlist.html:113]: Done

**Pull request merged**